.github/workflows/main.yml

```diff
@@ -8,7 +8,16 @@ on:
 jobs:
   phpunit:
     runs-on: ubuntu-latest
-    name: PHPUnit tests
+    strategy:
+      matrix:
+        include:
+          - drupal: '^8.9'
+            civicrm: '~5.32'
+          - drupal: '^8.9'
+            civicrm: '5.33.x-dev'
+          - drupal: '^9.0'
+            civicrm: '5.33.x-dev'
```

Comment:
Now we can test various versions of CiviCRM and Drupal minor releases much easier. We could port this to a new PR to just get `^8.9`  testing with 5.32 and 5.33RC

Reply:
That's really 👍 helpful!